x-pack/plugins/apm/public/components/app/error_group_details/distribution/index.tsx

```diff
@@ -88,7 +94,7 @@ export function ErrorDistribution({ distribution, title, fetchStatus }: Props) {
         <span>{title}</span>
       </EuiTitle>
       <ChartContainer
-        hasData={!!distribution}
+        hasData={!isEmpty}
```

Comment:
I don't think we need a new `isEmpty` prop on the component. What about 
```ts
const hasData = distribution.currentPeriod.length > 0;
```

Reply:
I though about that but what happens if we don't have anything for `currentPeriod` but we have data for `previosPeriod`? Maybe checking for those two?

Reply:
I removed the `isEmpty` property and instead allow the component to receive `distribution` optionally